argus/backend/plugins/core.py

```diff
@@ -27,7 +27,7 @@ class PluginModelBase(Model):
     _plugin_name = "unknown"
     # Metadata
     build_id = columns.Text(required=True, partition_key=True)
-    start_time = columns.DateTime(required=True, primary_key=True, clustering_order="DESC", default=datetime.now)
+    start_time = columns.DateTime(required=True, primary_key=True, clustering_order="DESC", default=datetime.now, custom_index=True)
```

Comment:
Does this add the index ? Or what this means ?

Reply:
This is probably an issue with CQL Engine - there's no index, I can execute this query through CQLsh just fine without Allow Filtering, but cqlengine complains while validating this query.